### ISSUE TITLE
eyre: cannot access Last-Event-ID parameter in eventsource

### ISSUE THREAD

**Issue by liam-fitzgerald** [urbit/urbit#4004]
We can't send custom headers on EventSource requests in the browser, so we can't use the channel resume functionality. We could ideally send this as a query parameter or something like that, so we can actually resume these.

**Comment by Fang-** [comment]
Relevant reading is https://github.com/whatwg/html/issues/2177#issuecomment-332071504 (and the rest of that thread). tldr "needs implementer interest" but browser vendors think they have better things to do. Recommended solution that doesn't modify the server-side is to use `fetch()`-based event streaming.

Worth looking into, I'd avoid doing non-standard things on the Eyre side if we can help it...

**Comment by tylershuster** [comment]
In my TS lib I've accessed `event.lastEventId` without problem. Is this something else?

**Comment by ohAitch** [comment]
I think this is access *for writing* when resuming connections. Which in theory the browser is supposed to do for you?

**Comment by liam-fitzgerald** [comment]
Yeah it's for resuming interrupted eventsources, instead of having to restart all subscriptions
